src/index.js

```diff
@@ -49,6 +51,33 @@ function tileReduce(options) {
   return ee;
 }
 
-function bboxTiles(bbox, zoom) {
-  return tilecover.tiles(bboxPolygon(bbox).geometry, {min_zoom: zoom, max_zoom: zoom});
+function parseToTiles(area, zoom) {
+  var jobArea = area,
+    poly = null;
+
+  if (new RegExp('.json').test(area) ||
+    new RegExp('.geojson').test(area)) {
+    var jobArea = JSON.parse(fs.readFileSync(area));
+  } 
+
+  if (jobArea instanceof Array) {
+    if (jobArea.length == 4 && typeof jobArea[0] === 'number') {
+      // bbox
+      poly = bboxPolygon(jobArea);
+    } else if (jobArea.length === 3 && typeof jobArea[0] === 'number') {
+      // [x,y,z] tile
+      poly = tilebelt.tileToGeoJSON(jobArea);
+      poly = turf.buffer(normalize(poly), -1, 'meters');
```

Comment:
@mourner that was leftover from grabbing the basis of this from tile-reduce-watchbot. I'm not exactly sure why you would really even need to do that. TBH, you should just `return [jobArea]` since it's already a tile